### PR TITLE
docs(): update new notifications toolbar

### PR DESCRIPTION
**Pull request by emoralesb05**
## Description

Add the new components or functions in the notifications toolbar.

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

### CODE REVIEW

File: src/app/components/toolbar/toolbar.component.ts [hunk 1]
```diff
@@ -16,7 +16,27 @@ import { getDirection, setDirection } from '../../utilities/direction';
...
+    icon: 'subject',
+    description: 'Render and navigate markdown',
+    title: 'Markdown Navigator'
+  }, {
+    route: '/components/functions',
+    icon: 'build',
+    description: 'New helper functions for multiple situations',
+    title: 'Functions'
+  }, {
+    rout: '/echarts/types/pie',
```
Comment (christianmemije) [inline src/app/components/toolbar/toolbar.component.ts]: typos

Comment (emoralesb05) [reply src/app/components/toolbar/toolbar.component.ts]: fixed

File: src/app/components/components/overview/overview.component.ts [hunk 2]
```diff
@@ -161,6 +161,12 @@ export class ComponentsOverviewComponent {
       route: 'flavored-markdown',
       title: 'Flavored Markdown Parser',
     },
+    {
```
Comment (christianmemije) [inline src/app/components/components/overview/overview.component.ts]: Should we add Functions to this page too?

Comment (emoralesb05) [reply src/app/components/components/overview/overview.component.ts]: I think so, since its supposed to be the overview. Ill add it

**Pull request merged**